app/services/scene-collections/nodes/scene-items.ts

```diff
@@ -139,18 +138,41 @@ export class SceneItemsNode extends Node<ISchema, {}> {
   load(context: IContext): Promise<void> {
     this.sanitizeIds();
 
-    this.data.items.forEach(item => {
-      if (item.sceneNodeType === 'item') {
-        if (item.streamVisible == null) item.streamVisible = true;
-        if (item.recordingVisible == null) item.recordingVisible = true;
+    // on first load, a dual output scene needs to assign displays and contexts to the scene items
+    // but if the scene item already has a display assigned, skip it
+    if (this.dualOutputService.views.hasNodeMap(context.scene.id)) {
+      // nodes must be assigned to a context, so if it doesn't exist, establish it
+      if (!this.videoSettingsService.contexts.vertical) {
+        this.videoSettingsService.establishVideoContext('vertical');
       }
-    });
 
-    if (
-      this.dualOutputService.views.hasNodeMap(context.scene.id) &&
-      !this.videoSettingsService.contexts.vertical
-    ) {
-      this.videoSettingsService.establishVideoContext('vertical');
+      const nodeMap = this.dualOutputService.views.sceneNodeMaps[context.scene.id];
+
+      const verticalNodeIds = Object.entries(nodeMap).map(([key, value]: [string, string]) => {
```

Comment:
i think you can just use `Object.values(nodeMap)` here

Reply:
Sure can lol